EpiAware/src/latent-processes.jl

```diff
@@ -14,12 +14,12 @@ end
     rw = Vector{T}(undef, n)
     ϵ_t ~ MvNormal(ones(n))
     σ²_RW ~ latent_process_priors.var_RW_dist
-    init_rw_value ~ latent_process_priors.init_rw_value_dist
+    init ~ latent_process_priors.init_rw_value_dist
     σ_RW = sqrt(σ²_RW)
 
-    rw[1] = init_rw_value + σ_RW * ϵ_t[1]
+    rw[1] = σ_RW * ϵ_t[1]
     for t = 2:n
         rw[t] = rw[t-1] + σ_RW * ϵ_t[t]
     end
-    return rw, (; σ_RW, init_rw_value, init = rw[1])
+    return rw, (; σ_RW, init)
```

Comment:
I can see why you have coupled initialisation like this to the random walk but in a more general setting it might not be the correct choice (where you have random walks that stop and start on top of some other latent process). I suggest we leave as is and address if and when it comes up (which I don't think it will).